website/content/docs/secrets/key-management/awskms.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: AWS KMS - Key Management - Secrets Engines
+sidebar_title: AWS KMS
+description: AWS KMS is a supported KMS provider of the Key Management secrets engine.
+---
+
+# AWS KMS
+
+~> **Note:** This provider is currently a **_beta_** feature and not recommended
+for deployment in production.
+
+The Key Management secrets engine supports lifecycle management of keys in [AWS KMS](https://aws.amazon.com/kms/)
+regions. This is accomplished by configuring a KMS provider resource with the `awskms` provider and
+other provider-specific parameter values.
+
+The following sections describe how to properly configure the KMS provider resource
+to enable the functionality.
+
+## Authentication
+
+The Key Management secrets engine must have sufficient permissions to manage keys in an AWS KMS
+region. The authentication parameters can be set in the following order of precedence: KMS provider
```

Comment:
nit: is it worth breaking out these orders of precedence into a list? I found it easy to skim over that information.

Reply:
Good idea. Will do 👍

Reply:
Done in a1a6715.